sdk/formrecognizer/ai-form-recognizer/src/index.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="../dom-shims.d.ts" />
```

Comment:
Almost all of our packages have the triple slash reference. Is it time to revisit the requirement of this linting rule?
cc @bterlson, @xirzec